components/ContactsItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from "react";
+
+interface Props {
+  title: string;
+  text: string;
+}
+
+export const ContactsItem: FC<Props> = ({ title, text, children }) => {
+  return (
+    <div className="contacts__item">
+      <div className="item__title">{title}</div>
```

Comment:
`contacts-item__title` instead `item__title`

`title` must be

- h1
- h2
- h3

..etc